fhttp/http.go

```diff
@@ -35,6 +35,8 @@ import (
 	// get /debug/pprof endpoints on default server - make sure report only doesn't have it
 	_ "net/http/pprof"
 
+	"sort"
```

Comment:
please put that import with the others (should be sorted automatically bu go fmt)

Reply:
Done